Mathlib/Tactic/Cases.lean

```diff
@@ -88,13 +88,13 @@ elab (name := induction') "induction' " tgts:(casesTarget,+)
       g.assign result.elimApp
       let subgoals ← ElimApp.evalNames elimInfo result.alts withArg
         (numGeneralized := fvarIds.size) (toClear := targetFVarIds)
-      setGoals (subgoals ++ result.others).toList
+      setGoals <| (subgoals ++ result.others).toList ++ gs
```

Comment:
This is the actual fix for #2016, right?

Reply:
Yes, but you can't write this line without the other line to define `gs`